pkg/resource/models.go

```diff
@@ -49,3 +50,19 @@ type OutResponse struct {
 	Version  Version    `json:"version"`
 	Metadata []Metadata `json:"metadata"`
 }
+type Headers map[string]string
+
+type LargeObjectOpts struct {
```

Comment:
Is this struct used anywhere?

Reply:
in the out.go

Reply:
In `out.go` I can only find `swift.LargeObjectOpts` which is a struct that is defined in the swift client package, no?